src/main/resources/l10n/JabRef_zh.properties

```diff
@@ -2278,6 +2284,13 @@ Full_text_document_download_failed_for_entry_%0.=
 Look_up_full_text_documents=
 You_are_about_to_look_up_full_text_documents_for_%0_entries.=
 last_four_nonpunctuation_characters_should_be_numerals=
+
+Author=
+Date=
+File_annotations=
+Show_file_annotations=
+Adobe_Acrobat_Reader=
+Sumatra_Reader=
```

Comment:
The name is Sumatra PDF (https://www.sumatrapdfreader.org/free-pdf-reader-de.html) and does not need to be translated, does it? - See asian Sumatra PDF page: https://www.sumatrapdfreader.org/free-pdf-reader-ja.html